### ISSUE TITLE
Progress feedback when deleting files etc

### ISSUE THREAD

**Issue by alexhunsley** [qarmin/czkawka#1088]
**Feature Description**

It would be nice to see some GUI feedback when deleting lots of files etc. Currently it just sort of sits there and you hope it's doing things.

**Comment by Jefferson-tan** [comment]
Agreed, some progress bar or a dialog that keeps the program alive would be nice. Currently when deleting very large files, it just sits there and Linux (I'm using Nobara with GNOME) keeps trying to kill it since it thinks that the program is unresponsive.

**Comment by qarmin** [comment]
It was already added in CLI/Krokiet
In Gtk version I treat this as feature, so it won't be implemented by me, because it is in maintenance mode
